### DISCUSSION TITLE
Requesting feature to support xterm screenReaderMode & xterm-addon-web-links together

### DISCUSSION THREAD

**Discussion by isubasin** [sourcelair/xterm.js#4477] [Ideas]
Hello @Tyriar & Team,  

Based off from the discussion https://github.com/xtermjs/xterm.js/discussions/4429 that @RahjIII reported, we would love to be able to have both these features working together.

Currently we have chosen the links over accessibility as you have suggested we have to chose one, but would like to enable screenReadyMode as well when possible. 

Therefore submitting this ticket hoping for a chance to have this fixed/implemented as a feature.

Thank you in advance! Appreciate your work & support as always...

**Comment by jerch** [comment]
Attempt to solve this is in: #4491, which suppresses pointer events from the accessibility div layer. Not sure yet, if this breaks screenreader apps though (it certainly will, if they rely on pointer events...)
